apps/examples/i2schar/i2schar_receiver.c

```diff
@@ -80,27 +80,32 @@
  *   This is the entry point for the receiver thread.
  *
  ****************************************************************************/
+FAR struct ap_buffer_s *g_apb[CONFIG_EXAMPLES_I2SCHAR_TXBUFFERS];
```

Comment:
The names suggests that these are references to TX buffers, but its used for RX & TX. Though this takes global space, this drastically cuts down the complexity by having a dynamic list. I vote for this.

Reply:
Indeed, it has to be changed to RX in other places also.